### PR TITLE
Fix for missing log in longreqs file after a long chunked transaction completes

### PR DESCRIPTION
**Pull request by aakash10292**
porting https://github.com/bloomberg/comdb2/pull/3400

### CODE REVIEW

File: db/sqlinterfaces.c [hunk 1]
```diff
@@ -1651,11 +1651,16 @@ int handle_sql_begin(struct sqlthdstate *thd, struct sqlclntstate *clnt,
                      enum trans_clntcomm sideeffects)
 {
     Pthread_mutex_lock(&clnt->wait_mutex);
-    /* if this is a new chunk, do not stop the hearbeats */
-    if (sideeffects != TRANS_CLNTCOMM_CHUNK)
+    /* if this is a new chunk, do not stop the hearbeats.*/
+    if (sideeffects != TRANS_CLNTCOMM_CHUNK){
```
Comment (riverszhang89) [inline db/sqlinterfaces.c:1655]: format is off.

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>3/456 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>socksql_master_swings<br>sc_parallel_logicalsc_generated<br>tunables</p>

**Review by nirbhayc — DISMISSED** [review]

**Review by mponomar — DISMISSED** [review]

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: <b>Error</b>. &#x26A0;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>3/456 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>socksql_master_swings<br>sc_truncate<br>triggersc_latency</p>

**Comment by riverszhang89** [comment]
Looks good. Can you please fix the format?

**Review by mponomar — APPROVED** [review]

**Review by mponomar — APPROVED** [review]

**Pull request merged**